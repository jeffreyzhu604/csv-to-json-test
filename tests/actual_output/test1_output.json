[
[,
{"fname":{"nickname":"Jeff"},"lname":"Zhu","age":"90","countryOfResidences":{"city":{"town":"Vancouver"}},"slogan":{"best":"Ayyy","worst":"haha"}},,
{"fname":{"nickname":"HoJun"},"lname":"Lee","age":"26","countryOfResidences":{"city":{"town":"Lougheed"}},"slogan":{"best":"Life's good","worst":"wompwomp"}},,
{"fname":{"nickname":"Jeff"},"lname":"Zhu","age":"90","countryOfResidences":{"city":{"town":"Vancouver"}},"slogan":{"best":"Ay;yy","worst":"ha;ha"}},,
{"fname":{"nickname":"HoJun"},"lname":"Lee","age":"26","countryOfResidences":{"city":{"town":"Lougheed"}},"slogan":{"best":"Life;s good","worst":"\"womp"},"field7":"womp\""},,
{"fname":{"nickname":"Jeff"},"lname":"Zhu","age":"90","countryOfResidences":{"city":{"town":"Vancouver"}},"slogan":{"best":"Ayyy","worst":"haha"}},,
{"fname":{"nickname":"HoJun"},"age":"26","countryOfResidences":{"city":{"town":"Lougheed"}},"slogan":{"best":"Life's good","worst":"wompwomp"}},,
{"lname":"Zhu","age":"90","countryOfResidences":{"city":{"town":"Lougheed"}},"slogan":{"best":"     Ayyy     ","worst":"haha"}},,
{"fname":{"nickname":"HoJun"},"age":"26","countryOfResidences":{"city":{"town":"Lougheed"}},"slogan":{"best":"     Life's good     ","worst":"wompwomp"}},,
{"fname":{"nickname":"null"},"lname":"Zhu","age":"90","countryOfResidences":{"city":{"town":"Vancouver"}},"slogan":{"best":"Ayyy","worst":"haha"}},,
{"fname":{"nickname":"HoJun"},"lname":"null","age":"26","countryOfResidences":{"city":{"town":"Lougheed"}},"slogan":{"best":" \"Life's good\" ","worst":"wompwomp"}},,
]
]
