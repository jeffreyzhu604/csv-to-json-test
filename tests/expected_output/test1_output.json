[
[,
{"field1":"fname.nickname","field2":"lname","field3":"age","field4":"countryOfResidence.city.town","field5":"slogan.best","field6":"slogan.worst"},,
{"field1":"Jeff","field2":"Zhu","field3":"90","field4":"Vancouver","field5":"\"Ayyy\"","field6":"\"haha\""},,
{"field1":"HoJun","field2":"Lee","field3":"26","field4":"Lougheed","field5":"\"Life's good\"","field6":"\"wompwomp\""},,
{"field1":"Jeff","field2":"Zhu","field3":"90","field4":"Vancouver","field5":"\"Ayyy\"","field6":"\"ha","field7":"ha\""},,
{"field1":"HoJun","field2":"Lee","field3":"26","field4":"Lougheed","field5":"Life;s good","field6":"\"wompwomp\""},,
{"field2":"Zhu","field3":"90","field4":"Vancouver","field5":"\"Ayyy\"","field6":"\"haha\""},,
{"field1":"HoJun","field3":"26","field4":"Lougheed","field5":"\"Life's good\"","field6":"\"wompwomp\""},,
{"field1":"Jeff","field2":"Zhu","field3":"90","field4":"Lougheed","field5":"\"     Ayyy     \"","field6":"\"haha\""},,
{"field1":"HoJun","field2":"Zhu","field3":"90","field4":"Lougheed","field5":"\"     Life's good     \"","field6":"\"wompwomp\""},,
{"field1":"Jeff","field2":"Zhu","field3":"\"90\"","field4":"Vancouver","field5":"\"Ayyy\"","field6":"\"haha\""},,
{"field1":"HoJun","field2":"Lee","field3":"26","field4":"Lougheed","field5":"\"/\"Life's good/\"\"","field6":"\"wompwomp\""},,
]
]
